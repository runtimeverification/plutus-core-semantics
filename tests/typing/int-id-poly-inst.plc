(program 1.0.0 { (abs s (size) (lam x [ (con integer) s ] x)) (con 3) })
