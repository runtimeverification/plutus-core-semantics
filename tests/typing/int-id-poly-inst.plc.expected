<generatedTop>
  <k>
    ( fun [ ( con integer ) ( con 3 ) ] [ ( con integer ) ( con 3 ) ] ) :: ( type )
  </k>
  <env>
    .
  </env>
</generatedTop>
